assets/js/app.js

```diff
@@ -18,6 +18,14 @@ import NProgress from "nprogress"
 import { LiveSocket } from "phoenix_live_view"
 import Hooks from "./hooks"
 
+window.rememberName = () => {
+    let name = document.getElementById("name").value
+    let xhr = new XMLHttpRequest()
```

Comment:
We can use the more modern fetch API here. 

https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API

Reply:
See, this is why I PRed this :) old habits

Reply:
Hehe, yes, doing this in my version of this… blogging it as we speak :D 

```
fetch(`/api/session?$foo=${encodeURIComponent(bar)}`, { method: "post" })
```